CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+# next (planned @ 2.1.0)
+- Adopts PSR-12 syntax everywhere (:heart: [jrzepa](https://github.com/jrzepa))
+
+## Breaking Changes
+- **Drops support for PHP 7.2** - With 0% adoption, we are dropping support for PHP 7.2. This allows us to use modern coding standards such as PSR-12, without the risk of breaking the CI environment. 7.2 may still work, but we strongly encourage you to update your PHP version.
```

Comment:
Just because no one is installing it on PHP 7.2, doesn't actually mean that no one is running it on PHP 7.2

Wikipedia sites (et al) all run this on PHP 7.2, but will be installed into our vendor repo using newer versions on dev machines :)

Reply:
That's useful context! I only had the packagist numbers as a reference point. I'll move this back into the discussion, but how would we like to say "works, but modern code standards mean CI isn't running against 7.2"